recipes-security/refpolicy/refpolicy-mcs-2.%/patches/xc-installer-interfaces.diff

```diff
@@ -1,10 +1,19 @@
-Index: refpolicy/policy/modules/system/lvm.te
-===================================================================
---- refpolicy.orig/policy/modules/system/lvm.te
-+++ refpolicy/policy/modules/system/lvm.te
+--- a/policy/modules/system/lvm.te
++++ b/policy/modules/system/lvm.te
 @@ -427,4 +427,5 @@ optional_policy(`
  	updatemgr_dontaudit_rw_fifo_files(lvm_t)
  	updatemgr_dontaudit_rw_stream_sockets(lvm_t)
  	updatemgr_dontaudit_search_storage(lvm_t)
 +	xc_installer_read_tmp_files(lvm_t)
```

Comment:
Why this change?  Only the `mounton` one is directly explained by the commit message.  Does an LVM tool poke at the partition after it is mounted?

Reply:
This is context change only due to `quilt refresh`. I am using the same quiltrc (see [wiki](https://openxt.atlassian.net/wiki/spaces/DC/pages/20905986/Dealing+with+quilt) for the OpenXT layer, so Index and the separation line are gone, and `-ab` is used.
`xc_installer_read_tmp_files(lvm_t)` is already applied earlier.

Reply:
Oh, I misread the diff of a diff again.  Sorry for the noise.